packages/core/types/ember-get-config/index.d.ts

```diff
@@ -1,6 +1,7 @@
 declare module 'ember-get-config' {
   import NaviConfig from 'navi-config';
   const value: {
+    modulePrefix: string;
     navi: NaviConfig;
   };
```

Comment:
Can we do this for all the `ember-get-config`s, also we can import the config directly so we don't have to duplicate the types
```suggestion
  import config from 'dummy/config/environment';
  const value: typeof config & { navi: NaviConfig };
```